griptape/processors/amazon_comprehend_processor.py

```diff
@@ -0,0 +1,31 @@
+from .base_processors import BasePromptStackProcessor
+import boto3
+
+
+class AmazonComprehendPiiProcessor(BasePromptStackProcessor):
+    def __init__(self, comprehend_client=None, custom_filter_func=None):
```

Comment:
Should be using `attrs` `field`. Take at something like `BedrockTitanTokenizer` for how we integrate `boto3.Session` and other fields.

Reply:
done, I also added some type hints for this class